compiler/luci/pass/src/FuseBCQPass.cpp

```diff
@@ -107,7 +122,11 @@ class BCQConverter final
   bool do_w_x(luci::CircleConst *node)
   {
     const auto prefix = node_name_prefix(node->name());
-    return _do_w_x[prefix]->at<loco::DataType::BOOL>(0);
+
+    if (_do_w_x[prefix]->dtype() == loco::DataType::S32)
+      return _do_w_x[prefix]->at<loco::DataType::S32>(0) == 1;
+    else
+      return _do_w_x[prefix]->at<loco::DataType::BOOL>(0);
```

Comment:
maybe adding an assert for `_do_w_x[prefix]->dtype()` be `loco::DataType::S32` or `loco::DataType::BOOL` ?